components/MembershipsWithData.js

```diff
@@ -91,8 +101,8 @@ class MembershipsWithData extends React.Component {
         </style>
 
         <div className="Collectives cardsList">
-          {memberships.map(membership => (
-            <Membership key={membership.id} membership={membership} LoggedInUser={LoggedInUser} />
+          {collectiveIds.map(id => (
```

Comment:
Is the goal of `collectiveIds` to preserve the original order?

Reply:
Yes, it is.

Reply:
Well thought :+1: